Library/Formula/wxmac.rb

```diff
@@ -69,6 +71,14 @@ def install
 
     system "./configure", *args
     system "make", "install"
+
+    if build.with? "static"
+      args.delete("--enable-shared")
+      args << "--disable-shared"
+
+      system "./configure", *args
+      system "make", "install"
```

Comment:
Unfortunately, `make install` does need to be called twice, otherwise only the static libraries get installed